mission_control/test/test_behavior_object.cpp

```diff
@@ -50,12 +52,13 @@ GTEST_TEST(BehaviorTest, TestIfBehaviorStopsAfterHalt)
   ASSERT_TRUE(dummyBehavior.isHalted());
 }
 
-GTEST_TEST(BehaviorTest, TestIfBehaviorChangeStatusToIdleAfterSuccess)
+GTEST_TEST(BehaviorTest, TestIfBehaviorDoesntChangeStatusToIdleAfterSuccess)
 {
   BT::NodeConfiguration nodeConfig;
   DummyBehavior dummyBehavior("dummy", nodeConfig);
   dummyBehavior.changeStatus(BT::NodeStatus::SUCCESS);
-  ASSERT_EQ(dummyBehavior.tick(), BT::NodeStatus::IDLE);
+  dummyBehavior.tick();
+  ASSERT_EQ(dummyBehavior.tick(), BT::NodeStatus::SUCCESS);
```

Comment:
@patapufate I'm not sure I understand the test.

Reply:
As commented above, if we tick over the same "Action" in SUCCESS of FAILURE state, we need to assure that the state will be the same.